### PR TITLE
Agent 3.66

### PR DESCRIPTION
**Pull request by moskyb**
This PR does two things:
- Updates the CLI reference to be generated from the latest version of the agent (the first commit)
- Adds new CLI reference entries for the recently-added `secret get` and `redactor add` commands

Note that if you have comments on the wording in generated files, i can't change them here, and we should work on submitting a PR to the agent to clear up wording there.

### CODE REVIEW

**Review by triarius — APPROVED** [review]
LGTM. I think we need to change our processes so that the person who made these changes (me) is prompted to add these, not the person who happens to be releasing the agent. Perhaps we should add a section to the PR template?

**Comment by moskyb** [comment]
@triarius good call - will add something to the agent PR template

File: pages/agent/v3/cli_secret.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# `buildkite-agent secret`
+
+> Note: The `secrets` command, as well as its associated server-side features, are currently in feature preview. If you're interested in trialing Buildkite Secrets, [contact support](mailto:support@buildkite.com).
```
Comment (gilesgas) [inline pages/agent/v3/cli_secret.md:3]: Thanks very much for your contribution @moskyb !

For consistency with the rest of the Buildkite Docs, I'd recommend sticking to our existing [style guide for callouts/admonitions](https://github.com/buildkite/docs/blob/main/styleguides/markdown-syntax-style.md#callouts):

```suggestion
> 📘 The `secrets` command, as well as its associated server-side features, are currently in feature preview.
> If you're interested in trialing Buildkite Secrets, [contact support](mailto:support@buildkite.com).
```

**Review by gilesgas — CHANGES_REQUESTED** [review]
Just one more comment (mine) to address ^

**Review by gilesgas — APPROVED** [review]

File: pages/agent/v3/cli_redactor.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# `buildkite-agent redactor`
+
+The Buildkite Agent automatically redacts some sensitive information from logs, such as secrets fetched with the [`secret get`](/docs/agent/v3/cli-secret) command, and any environment variables that match the value given in the [`--redacted-vars` flag](/docs/agent/v3/cli-start#redacted-vars).
+
+However, sometimes a job will source something sensitive through a side channel - perhaps a third-party secrets storage system like Hashicorp Vault or AWS Secrets Manager. In these cases, you can use the `redactor add` command to add the sensitive information to the redactor, ensuring it is redacted from subsequent logs.
```
Comment (gilesgas) [inline pages/agent/v3/cli_redactor.md:5]: One last thing, we're adopting Google's developer documentation guidance on using [em dashes to represent pauses in text like this (deliberately without spacing)](https://developers.google.com/style/dashes):

```suggestion
However, sometimes a job will source something sensitive through a side channel—perhaps a third-party secrets storage system like Hashicorp Vault or AWS Secrets Manager. In these cases, you can use the `redactor add` command to add the sensitive information to the redactor, ensuring it is redacted from subsequent logs.
```

Would be nice to commit this before merging :-)

Comment (moskyb) [reply pages/agent/v3/cli_redactor.md:5]: ah shit, just barely too slow 😅

Comment (gilesgas) [reply pages/agent/v3/cli_redactor.md:5]: Whoops, too late—my bad lol (another time)

**Pull request merged**